addons/l10n_hr_edi/__init__.py

```diff
@@ -0,0 +1,14 @@
+from . import models
+from . import wizard
+
+
+def post_init(env):
+    # Loading new field 'l10n_hr_tax_category_id' for existing Croatian taxes
+    for company in env['res.company'].search([('chart_template', '=', 'hr')], order="parent_path"):
+        env['account.chart.template'].with_company(company)._load_data({
+            'account.tax': {
+                xmlid: vals
+                for xmlid, vals in env['account.chart.template']._parse_csv('hr', 'account.tax', module='l10n_hr_edi').items()
+                if env['account.chart.template'].with_company(company).ref(xmlid, raise_if_not_found=True)
+            }
+        })
```

Comment:
The chart template can be reused, and the new method `_get_hr_edi_account_tax` as well. And the point of filtering is to only update existing records, why is `raise_if_not_found=True` used?
```suggestion
        Template = env['account.chart.template'].with_company(company)
        Template._load_data({
            'account.tax': {
                xmlid: vals
                for xmlid, vals in Template._get_hr_edi_account_tax().items()
                if Template.ref(xmlid, raise_if_not_found=False)
            }
        })
```

Reply:
Thanks so much - I think the `raise_if_not_found` should definitely be False as you say :smile:

Reply:
Ahhhh :hushed: I knew about it haha but I didn't think it would be useful here! Good to know! :smile: